src/app/components/SearchResults/SearchResultsPage.jsx

```diff
@@ -15,6 +13,35 @@ import TotalWorks from '../SearchForm/TotalWorks';
 
 import featureFlagConfig from '../../../../featureFlagConfig';
 import config from '../../../../appConfig';
+import SearchHeader from '../SearchForm/SearchHeader';
+import SearchResults from './SearchResults';
+
+export const loadSearch = (props) => {
```

Comment:
Why move this out of the component? It doesn't seem to be used anywhere else, and I don't think it will be? Not that it doesn't work, but just curious about the move.

Reply:
I think I'd moved it out originally to test it in isolation, but it can also be tested using `wrapper.instance()`

Reply:
If it makes testing easier/clearer that's fine by me, just wondering what the reason was

Reply:
tbh I'm still waffling on how best to organize the code so that everything can get tested in a sane way... 
so it might get moved back, but for now it doesn't seem like it's causing harm where it is